### PR TITLE
responsive-calculator

### PR DESCRIPTION
**Pull request by Saminsfw**
calculator-desktop
calculator-tab
calculator-mobile

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+let input = document.getElementById("inputBox");
+let buttons = document.querySelectorAll("button");
+let result = input.value;
+let buttonArray = Array.from(buttons);
+buttonArray.forEach((button) => {
+  button.addEventListener("click", (clickedButton) => {
+    if (clickedButton.target.innerHTML == "=") {
+      result = eval(result);
+      input.value = result;
```
Comment (UmmarHamid) [inline script.js]:
```suggestion
      input.value = eval(result);
```

File: script.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    } else if (clickedButton.target.innerHTML == "AC") {
+      result = "";
+      input.value = result;
```
Comment (UmmarHamid) [inline script.js]:
```suggestion
      input.value = "";
```

File: script.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    } else if (clickedButton.target.innerHTML == "DEL") {
+      result = result.substring(0, result.length - 1);
+      input.value = result;
+    } else {
+      result += clickedButton.target.innerHTML;
+      input.value = result;
```
Comment (UmmarHamid) [inline script.js]:
```suggestion
    } else if (clickedButton.target.innerHTML == "DEL") {
       input.value  = input.value.substring(0, input.value.length - 1);
    } else {
      input.value += clickedButton.target.innerHTML;
```

**Review by UmmarHamid — CHANGES_REQUESTED** [review]
Looks good.
Just some optimizations

**Comment by UmmarHamid** [comment]
> _ No description provided. _

@Saminsfw I cannot find a proper screenshot to review this at 3 breakpoints. Can you add those please??

**Comment by Saminsfw** [comment]
calculator-desktop

**Comment by Saminsfw** [comment]
calculator-tab

**Comment by Saminsfw** [comment]
calculator-mobile

**Pull request merged**